packages/carousel/src/Carousel.js

```diff
@@ -1,18 +1,17 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
-import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
-import moize from 'moize'
-import { v4 as uuidv4 } from 'uuid'
-
+import { CarouselProvider, Slide } from 'pure-react-carousel'
 import { layoutToFlexWidths } from './layoutToFlexWidths'
 import { CarouselWrapper } from './Carousel.styles'
-
-const getSlideKey = moize(uuidv4)
+import { Flex, Relative } from 'pcln-design-system'
+import { Dots } from './Dots/Dots'
+import { ArrowButton } from './ArrowButton/ArrowButton'
+import { Slider } from './Slider/Slider'
```

Comment:
if you export these from an `index.js` file then you don't need the duplication in the import path

Reply:
I hate index files haha, but I can add those in

Reply:
Updated :)